content/docs/reference/glossary.md

```diff
@@ -282,10 +338,22 @@ A preview period during which users can try Neon.
 
 An encrypted access token that enables you to authenticate with Neon using the Neon API. An access token is generated when creating a Neon API key. For more information, see [Manage API keys](https://neon.tech/docs/get-started-with-neon/using-api-keys/).
 
+### tmpfs
+
+A temporary file storage system that uses a portion of a system's RAM to store files, improving performance by reducing disk usage.
+
 ### User
 
 See [Neon user](#neon-user) and [PostgreSQL role](#postgresql-role).
 
+### vm-informant
+
+A program that runs inside the VM alongside PostgreSQL, responsible for requesting more resources from the autoscaler-agent and validating proposed downscaling to ensure sufficient memory.
+
+### vCPU
+
+Virtual Central Processing Unit, a unit of processing power allocated to a virtual machine or compute instance.
```

Comment:
It may be helpful to define this in terms of `CPU`, instead of _fully_ expanding the acronym

Reply:
Changed as suggested.